src/workflow_review.md

```diff
@@ -0,0 +1,128 @@
+---
+description: "Git workflow for collaboration"
+title: Workflow Collaboration
+output:
+  html_document:
+    toc: true
+    toc_depth: 2
+    toc_float: true
+---
+
+These guidelines explain the different steps to be taken when collaborating
+in a Github repository while developing code in RStudio. The main aim is to
+give a concise overview rather than giving detailed information on different
+options or showing screenshots for each step. The steps presented here follow
+more or less [this schedule](https://guides.github.com/introduction/flow/) but
+the explanations and tips are rather complementary.
+
+# Add an issue
+
+Ask for changes or propose to make changes in a Github repository:
+
+- on the main page of the git repository on GitHub, hit tab `Issues`
+- check for similar issues (using the search function)
+    - if the issue already exists, add your comments or suggestions in this issue, or add a thumb emoticon (to follow the issue)
+- if your issue is not yet reported, hit the button `New issue`
+- give it an informative title and add a clear and complete description (with reproducible example if appropriate)
+
+Issues can be used to:
+
+- report to do's to yourself/team
+- propose and discuss additions/changes
+- report bugs in packages you're using
+
+Issues can be:
+
+- assigned to someone
+- labeled (e.g. bug, documentation, help wanted,...)
+
+# Add code in a new feature branch
+
+(If you don't have the project locally, check the manual [here](https://inbo.github.io/git-course/course_rstudio.html#23_clone_a_repo_to_work_locally).)
+
+In RStudio on the `Git`tab:
+
+- make sure the `master` branch is active and up to date (`Pull`)
+- add a `New Branch` (hit button) and give it an informative name
+- add new code or change code, and make a `Commit` for every change
+    - you can select specific lines or chunks, you don't have to commit all changes at once
+    - make 1 commit for each change and add a clear commit message
+- regularly `Push` your changes to the repository on github
```

Comment:
don't push too frequently as every push might trigger the CI system. I'd recommend to push at least once a day. And more frequent if needed.

Reply:
solved in a232cf5